### ISSUE TITLE
Accessories not persisted warning

### ISSUE THREAD

**Issue by ohmantics** [quentinms/homebridge-aranet4#11]
**Describe The Bug:**
Warning emitted during startup and no values reported

**To Reproduce:**

1. Install aranet4 plugin
2. Increase Bluetooth setup timeout to 30s
3. Increase device scan timeout to 30s
4. Warning emitted during startup
5. Device appears in HomeKit, but all values are 0.

**Expected behavior:**

No warning and device reports properly

**Logs:**

```[4/18/2022, 8:20:11 PM] [Aranet4] Adding new accessory: Aranet4
[4/18/2022, 8:20:11 PM] Failed to save cached accessories to disk: Converting circular structure to JSON
    --> starting at object with constructor 'Peripheral'
    |     property '_noble' -> object with constructor 'Noble'
    |     property '_peripherals' -> object with constructor 'Object'
    --- property 'f5ddf8329406' closes the circle
[4/18/2022, 8:20:11 PM] Your accessories will not persist between restarts until this issue is resolved.
```

**Plugin Config:**

```json
        {
            "name": "Aranet4",
            "co2AlertThreshold": 1400,
            "batteryAlertThreshold": 10,
            "sensorDataRefreshInterval": 300,
            "bluetoothReadyTimeout": 30,
            "bluetoothDeviceSearchTimeout": 30,
            "platform": "Aranet4"
        }
```

* **Plugin Version**: homebridge-aranet4 v0.0.1
* **Homebridge Version**: 1.4.0
* **Node.js Version**: v17.8.0
* **NPM Version**: 8.5.5
* **Operating System**: Raspbian 5.15.30-v7+ #1536

**Comment by quentinms** [comment]
Thanks for the detailed bug report! 
I can reproduce on my end, after deleting the existing accessory. 
I'll have a look.

**Comment by quentinms** [comment]
@ohmantics I believe the issue is fixed with `v0.0.2`. 
Please let me know if it now works for you too.

> 5. Device appears in HomeKit, but all values are 0.

This is possibly because the [default refresh value is 5m](https://github.com/quentinms/homebridge-aranet4/blob/7297a928c75912ede4a12259dc00b3b359f65b26/config.schema.json#L37) to preserve the battery, so it might stay at 0 for a little while. 

You can lower it and data should appear sooner.
